src/test/mir-opt/array_index_is_temporary.main.SimplifyCfg-elaborate-drops.after.mir.32bit

```diff
@@ -25,9 +25,9 @@ fn main() -> () {
 
     bb0: {
         StorageLive(_1);                 // scope 0 at $DIR/array-index-is-temporary.rs:13:9: 13:14
-        _1 = [const 42_u32, const 43_u32, const 44_u32]; // scope 0 at $DIR/array-index-is-temporary.rs:13:17: 13:29
+        _1 = [42_u32, 43_u32, 44_u32];   // scope 0 at $DIR/array-index-is-temporary.rs:13:17: 13:29
```

Comment:
Similarily here, though I am not sure if the `Constant` could be printed as an array, or would be printed as a raw allocation.

Reply:
it can be printed as an array, so yea, there would be no visible difference in the mir

Reply:
I'm wondering whether the information `const` vs `Aggregate` is useful, but the original comment by me that motivated @lzutao  to open this PR was solely for `FnDef`, where the `const` is actively confusing (at least in the position for function calls. Maybe we should just special case function calls to not print a `const`?

Reply:
I don't feel super strongly but I would prefer if this was only done for `FnDef`s tight now.